tests/E2E/test/campaigns/common_scenarios/order.js

```diff
@@ -200,8 +200,10 @@ module.exports = {
   checkExportedFile: async function (client) {
     await client.downloadCart(ShoppingCart.export_carts_button);
     await client.checkFile(global.downloadsFolderPath, global.exportCartFileName);
-    await client.readFile(global.downloadsFolderPath, global.exportCartFileName, 1000);
-    await client.checkExportedFileInfo(1000);
+    if (existingFile) {
```

Comment:
use global.existingFile instead

Reply:
Hi @PierreRambaud,
yes, it's the same, the existingFile here calls the global.existingFile parameter.

Thanks!

Reply:
it is not really the same thing since you can have a scoping variable named existingFile :/ with global, you are sure it will always call the good one :)

Reply:
yes  if we will add a scoping parameter with the same name existingFile, it will not take the global existingFile value for sure.

I will fix it so :) .

Thank you!